website/docs/reference/project-configs/query-comment.md

```diff
@@ -51,7 +51,7 @@ create view analytics.analytics.orders as (
 
 ## Using the dictionary syntax
 The dictionary syntax includes two keys:
-  * `comment` (optional, see above for default): The string to be injected to a query as a comment.
+  * `comment` (optional, see above for default): The string to be injected into a query as a comment.
```

Comment:
```suggestion
  * `comment` (optional, for more information, refer to the [default](/reference/project-configs/query-comment#default) section): The string to be injected into a query as a comment.
```

Reply:
Hiya @mirnawong1 

This is a non-blocker, but I thought it might be good to link out "default" in case some users weren't paying attention to the Default section or scrolled all to the top of the page.

Kind Regards
Natalie